conda_recipes/aftereffects-25.0/recipe/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set name = "aftereffects" %}
+{% set version_partial = "25" %}
+{% set version_minor = "0" %}
+{% set version = version_partial + "." + version_minor %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+- url: archive_files/Adobe_AfterEffects_{{ version.replace(".", "_") }}_Windows_installation.zip # [win64]
+  sha256: f181bddce95e0f45555c5188a9a515a43ce20d0071d566286f748cbba36b5716 # [win64]
```

Comment:
I'm assuming this would be replaced with the hash that customers generate?

Reply:
The hash should be the same for the same version of the archive.